llmfoundry/callbacks/curriculum_learning_callback.py

```diff
@@ -1,16 +1,13 @@
-# Copyright 2022 MosaicML LLM Foundry authors
+# Copyright 2024 MosaicML LLM Foundry authors
 # SPDX-License-Identifier: Apache-2.0
 
-"""Enable curriculum learning by resuming with a different dataset.
-
-This callback is currently experimental. The API may change without warning in
-the future.
-"""
+"""Enable curriculum learning by specifying a schedule of datasets to train
+on."""
```

Comment:
minor nit: new line for comment with one word is kinda bothersome

Reply:
Done